pkg/virt-controller/watch/drain/evacuation/evacuation.go

```diff
@@ -328,16 +318,36 @@ func (c *EvacuationController) execute(key string) error {
 	}
 
 	return c.sync(node, vmis, migrations)
+}
 
+func getEvictedVMIs(vmis []*virtv1.VirtualMachineInstance) []*virtv1.VirtualMachineInstance {
```

Comment:
This function name mislead me. Can I suggest to rename it to something like `VMIsToEvict`.